action.yml

```diff
@@ -61,7 +61,7 @@ runs:
       shell: bash
       run: |
         # Generate console credential file from env vars
-        echo "{\"consoleCredentials\":{\"audience\":\"https://axis.snxd.com/\",\"clientId\":\"${{ inputs.solsta_client_id }}\",\"clientSecret\":\"${{ inputs.solsta_client_secret }}\",\"grant\":\"clientCredentials\",\"scope\":\"d4tv1\"}}"  > client_credentials.json
+        echo "{\"consoleCredentials\":{\"audience\":\"https://axis.snxd.com/\",\"clientId\":\"${{ inputs.solsta_client_id }}\",\"clientSecret\":\"${{ inputs.solsta_client_secret }}\",\"grant\":\"clientCredentials\",\"scope\":\"\"}}"  > client_credentials.json
```

Comment:
Strange that scope is empty string. Perhaps @steve-tucker will have input on it.

Reply:
I think it should probably be removed.

Reply:
I will do so right now.  
Also, a Question: It would work the same either way?  IOW, is this change cosmetic or functional?

Reply:
tbh mostly undefined behavior, but there are tinges of cosmetic. I know what the behavior for a request with a missing scope is, I know what it is for a filled in scope. But I'm not sure what the oauth behavior for scope being an empty string is supposed to be. If you look it up and tell me it's defined in the specification, then I'll be satisfied also.